css/styles.css

```diff
@@ -386,3 +388,100 @@ button.form_btn:hover {
   grid-template-rows: 25% 25% 25% 25%;
   row-gap: 20px;
 }
+
+.contenido-form {
+  display: flex;
+}
+
+/*      Desktop Style     */
+
+@media (min-width: 768px) {
+  .portfolio {
+    display: flex;
+    display: none;
+  }
+
+  .portfolio1 {
+    display: flex;
+    width: auto;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .project-detail {
+    width: 569px;
+  }
+
+  .contenido1 {
+    display: flex;
+    width: 50%;
```

Comment:
- [ ] Please stick to the Figma template relate to the size of each column. The `contenido1` must use 40% of the width. 
- [ ] Kindly fix the styling of the technologies (Javascript, HTML, and CSS) presented in the image below.
Your app:
<img width="1280" alt="Screen Shot 2022-08-27 at 12 00 18 AM" src="https://user-images.githubusercontent.com/47673311/187011957-aa108239-52fd-42a0-ab30-536105c4e66f.png">

Figma template:
<img width="673" alt="Screen Shot 2022-08-27 at 12 00 51 AM" src="https://user-images.githubusercontent.com/47673311/187011968-61dc83e4-b488-408f-9066-d2598fad641b.png">